.github/workflows/R-CMD-check.yaml

```diff
@@ -73,6 +73,8 @@ jobs:
           extra-packages: |
             any::rcmdcheck
             bookdown=?ignore-before-r=3.5.0
+            patrick=?ignore-before-r=3.5.0
```

Comment:
Because `{patrick}` also depends on `{purrr}`:           

```r
> pak::pkg_deps_explain("lintr", "purrr", dependencies = "all")
✔ Updated metadata database: 5.10 MB in 12 files.                         
✔ Updating metadata database ... done                                     
lintr -> patrick -> purrr                                                  
lintr -> pkgdown -> purrr
```

Reply:
Cool I didn't know of this feature before 👍🏻